src/main/java/io/aiven/kafka/connect/common/config/FormatType.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.config;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public enum FormatType {
+
+    CSV("csv"),
+    JSONL("jsonl");
+
+    public static final String SUPPORTED_FORMAT_TYPES =
+            FormatType.names().stream()
+                    .map(c -> String.format("'%s'", c))
+                    .collect(Collectors.joining(", "));
+
+    public final String name;
+
+    private FormatType(final String name) {
+        this.name = name;
+
```

Comment:
Empty spaces ... What are we living for? :-D

Reply:
no need to rush

Reply:
:+1: 
- [ ] Remove empty spaces

Reply:
Well...Another one(commit) bites the dust ))))